Riot/Modules/Spaces/SpaceDetail/SpaceDetailPresenter.swift

```diff
@@ -56,7 +55,22 @@ class SpaceDetailPresenter: NSObject {
         self.viewModel.coordinatorDelegate = self
         self.presentingViewController = viewController
         self.sourceView = sourceView
-        self.selectedSpace = session.spaceService.getSpace(withId: spaceId)
+        
+        self.show(with: session)
+    }
+    
+    @objc func present(forSpaceWithSummary spaceSummary: MXPublicRoom,
```

Comment:
Is `forSpaceWithSummary spaceSummary` the right definition as the parameter is `MXPublicRoom `?